tests/async/tasyncawait_cyclebreaker.nim

```diff
@@ -65,7 +65,7 @@ while true:
   GC_collectZct()
 
   let (allocs, deallocs) = getMemCounters()
-  doAssert allocs < deallocs + 1000
+  doAssert allocs < deallocs + 150_000 # changed after await_template, not sure if it isn't too much
```

Comment:
We need to find the cause for this though. A refactoring shouldn't have this visible change in allocation behaviour.

Reply:
yes! I agree

Reply:
another change seems to lead to normal behavior